tests/test_cli/tools_for_testing.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""Tools used for CLI registry testing."""
+
+
+class AgentConfigMock:
+    """An object to mock Agent config."""
+
+    registry_path = 'registry'
+    name = 'name'
+
+
+class ContextMock:
+    """An object to mock Click Context."""
+
+    cwd = 'cwd'
+    agent_config = AgentConfigMock()
+
+
+class PublicIdMock:
+    """An object to mock PublicId."""
+
+    def __init__(self, name='name', author='author', version='1.0.0'):
```

Comment:
please everywhere where you use verison, import DEFAULT_VERSION from configurations base